### DISCUSSION TITLE
Webhooks don't pass email address

### DISCUSSION THREAD

**Discussion by mcconnon12** [swellstores/community#365] [Ideas]
Can we get the webhooks to pass the email address? I know you allow via Zapier app for orders but it would be extremely helpful to have the email address be passed through webhooks.

**Comment by igoraugustynski** [comment]
What I do is I get the order using swell-node, then get the email address from the order.

**Reply by mcconnon12** [reply]
Is there a way to do this via Pabbly Connect?

**Reply by ericingram** [reply]
Why not just fetch the email from the webhook handler?

**Reply by mcconnon12** [reply]
When I try to get it from webhook it does not come through. I've Zapier webhook, Pabbly, and about a half dozen others... the email does not come through. I've confirmed this with multiple folks at Swell that the email does not pass through which is extremely odd.

**Reply by ericingram** [reply]
That’s because email is not part of the cart/order object, but is a reference you can fetch. So why not fetch it?

**Reply by mcconnon12** [reply]
Is there a way I can do that via Pabbly when an order comes in?

**Reply by ericingram** [reply]
I'm not really familiar with Pabbly, but the upcoming App framework should make this a whole lot easier, since you'll be able to write code hosted by Swell to integrate with any other system.
